Fitness/Models/ProRoutine.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  RoutineType.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 3/2/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import Foundation
+
+struct ProRoutine {
+    enum ProRoutineType {
```

Comment:
Same here. This name could make better use of the current namespace.

Reply:
I'll change it to `RoutineType`